src/rf/js/src/core/models.js

```diff
@@ -60,7 +60,7 @@ var Layer = Backbone.Model.extend({
         return _.findWhere(this.get('images'), {id: id});
     },
 
-    isUploading: function() {
+    isPreValidated: function() {
```

Comment:
I think preValidated is better because it's the same length, but also indicates that the layer is before the point where it is validated.